### PR TITLE
App info

### PR DESCRIPTION
**Pull request by areedv**
Suggesting framework more than content. This pr adds a new package (shinyalert) to qmongr. Hence, qmongr-base-r needs to be rebuilt. Until then, tests will fail

### CODE REVIEW

**Comment by areedv** [comment]
The looks:
image

**Comment by yte0** [comment]
looks great:-)

File: R/app_info.R [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#' @export
+#' @examples
+#' version_info()
+
+version_info <- function(newline = "<br>") {
+  
+  conf <- get_config()
+  pkg <- conf$app_text$info$version$app
+  m <-installed.packages()[pkg, c(1,3)]
+  paste0(pkg, " v", m[pkg, "Version"], newline, collapse = "")
```
Comment (arnfinn) [inline R/app_info.R]: Is `collapse` needed?

File: R/mod_quality_overview.R [hunk 2]
```diff
@@ -227,6 +228,23 @@ mod_quality_overview_server <- function(input,
         sort(decreasing = T)
     )
   })
+
+  output$appInfo <- shiny::renderUI({
```
Comment (arnfinn) [inline R/mod_quality_overview.R]: Use snake_case instead since the rest of the outputs are using snake_case? `app_info`

**Comment by arnfinn** [comment]
Update NEWS.md

File: R/mod_quality_overview.R [hunk 3]
```diff
@@ -227,6 +228,23 @@ mod_quality_overview_server <- function(input,
...
+    )
+  })
+  shiny::observeEvent(input$app_info, {
+    shinyalert::shinyalert(title = config$app_text$info$title,
+                           text = qmongr::version_info(),
+                           type = "",
+                           closeOnEsc = TRUE, closeOnClickOutside = TRUE,
+                           html = TRUE,
+                           confirmButtonText = qmongr::no_opt_out_ok())
+  })
```
Comment (arnfinn) [inline R/mod_quality_overview.R]: Is it possible to test this part of code?

File: R/mod_quality_overview.R (hunk 2 above)
Comment (areedv) [reply R/mod_quality_overview.R]: trying my best to adapt...

File: R/app_info.R (hunk 1 above)
Comment (areedv) [reply R/app_info.R]: Sure. Function should provide only one string (not several)

Comment (arnfinn) [reply R/app_info.R]: ok:)

File: R/mod_quality_overview.R (hunk 3 above)
Comment (areedv) [reply R/mod_quality_overview.R]: I'll try

Comment (areedv) [reply R/mod_quality_overview.R]: superficially, at least

File: NEWS.md [hunk 4]
```diff
@@ -4,6 +4,10 @@
 
 * Restructured `mod_quality_overview_server` tests.
 
+# qmongr 0.6.0
+
+* Added information widget ([#90](https://github.com/SKDE-Felles/qmongr/pull/90))
+
 # qmongr 0.5.0
```
Comment (arnfinn) [inline NEWS.md]: The head of `NEWS.md` is now:
```markdown
# Unreleased

## Internal

* Restructured `mod_quality_overview_server` tests.

# qmongr 0.6.0

* Added information widget ([#90](https://github.com/SKDE-Felles/qmongr/pull/90))
```
I would prefer
```markdown
# Unreleased

## New features

* Added information widget ([#90](https://github.com/SKDE-Felles/qmongr/pull/90))

## Internal

* Restructured `mod_quality_overview_server` tests.
```
or (if you prefer):
```markdown
# qmongr 0.6.0

## New features

* Added information widget ([#90](https://github.com/SKDE-Felles/qmongr/pull/90))

## Internal

* Restructured `mod_quality_overview_server` tests.
```

**Comment by areedv** [comment]
Merge will be closing #69 even though more content is to be added later

File: DESCRIPTION [hunk 5]
```diff
@@ -1,6 +1,6 @@
 Package: qmongr
 Title: Lightweight Quality Assessment of Norwegian Hospitals 
-Version: 0.5.0.9000
+Version: 0.6.0
```
Comment (arnfinn) [inline DESCRIPTION]: Should we do this directly on the master branch afterwards? How it is done in 04698d380c340

Comment (arnfinn) [reply DESCRIPTION]: At this stage it is probably OK to do it as you did it here.

**Pull request merged**